src/plugins/visualize/public/application/utils/use/use_vis_byvalue.ts

```diff
@@ -41,7 +41,7 @@ export const useVisByValue = (
   useEffect(() => {
     const { chrome } = services;
     const getVisInstance = async () => {
-      if (!valueInput || loaded.current) {
+      if (!valueInput || loaded.current || !visEditorRef.current) {
```

Comment:
@ThomThomson Could you check if this is still working as expected? This should not break things theoretically, because the `Editor` would anyway not have worked if there was no HTML element passed into, just want to double check.